pool/tick_bitmap.gno

```diff
@@ -31,11 +30,9 @@ func (pool *Pool) tickBitmapFlipTick(
 	}
 
 	wordPos, bitPos := tickBitmapPosition(tick / tickSpacing)
+	pool.tickBitmaps[wordPos] = pool.tickBitmaps[wordPos].NilToZero()
 
-	// 1 << bitPos
 	mask := new(u256.Uint).Lsh(u256.NewUint(1), uint(bitPos))
-
-	pool.tickBitmaps[wordPos] = pool.tickBitmaps[wordPos].NilToZero()
 	pool.tickBitmaps[wordPos] = new(u256.Uint).Xor(pool.tickBitmaps[wordPos], mask)
 }
 
```

Comment:
Improve nested logic in `tickBitmapNextInitializedTickWithInOneWord`. Break them as much as possible and divide them into other func if necessary

Reply:
22d47515d930c650b5e6f43b8351b39bea1eb613